paypal/express_checkout/models.py

```diff
@@ -20,7 +20,7 @@ class ExpressCheckoutTransaction(models.Model):
     currency = models.CharField(max_length=8, null=True, blank=True)
 
     CREATED, SAVED, APPROVED, VOIDED, COMPLETED = 'CREATED', 'SAVED', 'APPROVED', 'VOIDED', 'COMPLETED'
-    status = models.CharField(max_length=8)
+    status = models.CharField(max_length=9)
```

Comment:
Please generate a migration for this change - this needs to be added to `paypal/migrations`.